sdk/src/network.rs

```diff
@@ -5,22 +5,17 @@ use fvm_shared::econ::TokenAmount;
 use fvm_shared::version::NetworkVersion;
 
 use crate::sys;
+use crate::vm::MESSAGE_DETAILS;
```

Comment:
Can we unify the naming here? Should this be Invocation Context (how it's called kernel side)?